docs/linux.md

```diff
@@ -47,12 +47,12 @@ Device found:
 
 ###### Running the `gst-launch-1.0` command to start streaming from a RTSP camera source.
 ```
-$ gst-launch-1.0 -v rtspsrc location=rtsp://YourCameraRtspUrl short-header=TRUE ! rtph264depay ! video/x-h264, format=avc,alignment=au ! h264parse ! kvssink stream-name=YourStreamName storage-size=128
+$ gst-launch-1.0 -v rtspsrc location=rtsp://YourCameraRtspUrl short-header=TRUE ! rtph264depay ! h264parse ! h264parse ! kvssink stream-name=YourStreamName storage-size=128
```

Comment:
I do not understand why the `h264parse` element is there twice that isn't right.  I think I see the bug though the `h264parse` element needs to come AFTER `rtph264depay` and BEFORE `video/x-h264, format=avc,alignment=au`

We should definitely not have two consecutive `h264parse` elements.